### DISCUSSION TITLE
I would like to change the manifest.jon file of the web client.

### DISCUSSION THREAD

**Discussion by nascin** [flet-dev/flet#201] [Q&A]
How to change flet client settings? I would like to change the manifest.jon file of the web client.

**Comment by keyiflerolsun** [comment]
Like my sample repo ; 

https://github.com/keyiflerolsun/KekikHava

Preview : https://kekikhava.keyiflerolsun.repl.co/#/

**Reply by nascin** [reply]
Interesting, I will definitely look into the possibilities. Thanks.
